client/src/components/Community/Community.css

```diff
@@ -20,12 +20,10 @@
 }
 
 .community-component .title {
-  opacity: 0;
-  width: 0;
-  height: 0;
-  pointer-events: none;
-  margin: 0;
-  padding: 0;
```

Comment:
Retain these properties.

Reply:
I have made some changes on the branch. I keep these properties, but also add some new properties.